pkg/asset/agent/installconfig.go

```diff
@@ -313,48 +333,16 @@ func warnUnusedConfig(installConfig *types.InstallConfig) {
 			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ExternalBridge))
 		}
-		if baremetal.ProvisioningNetwork != defaultBM.ProvisioningNetwork {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningNetwork")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningNetwork))
-		}
-		if baremetal.ProvisioningBridge != defaultBM.ProvisioningBridge {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningBridge")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningBridge))
-		}
-		if baremetal.ProvisioningNetworkInterface != defaultBM.ProvisioningNetworkInterface {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningNetworkInterface")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningNetworkInterface))
-		}
-		if baremetal.ProvisioningNetworkCIDR.String() != defaultBM.ProvisioningNetworkCIDR.String() {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningNetworkCIDR")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningNetworkCIDR))
-		}
 		if baremetal.DeprecatedProvisioningDHCPExternal != defaultBM.DeprecatedProvisioningDHCPExternal {
 			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningDHCPExternal")
 			logrus.Warnf(fmt.Sprintf("%s: true is ignored", fieldPath))
 		}
-		if baremetal.ProvisioningDHCPRange != defaultBM.ProvisioningDHCPRange {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningDHCPRange")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningDHCPRange))
+		if baremetal.ProvisioningBridge != defaultBM.ProvisioningBridge {
+			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningBridge")
+			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningBridge))
 		}
 
 		for i, host := range baremetal.Hosts {
-			if host.BMC.Username != "" {
-				fieldPath := field.NewPath("Platform", "Baremetal", fmt.Sprintf("Hosts[%d]", i), "BMC", "Username")
-				logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, host.BMC.Username))
-			}
-			if host.BMC.Password != "" {
-				fieldPath := field.NewPath("Platform", "Baremetal", fmt.Sprintf("Hosts[%d]", i), "BMC", "Password")
-				logrus.Warnf(fmt.Sprintf("%s is ignored", fieldPath))
-			}
-			if host.BMC.Address != "" {
-				fieldPath := field.NewPath("Platform", "Baremetal", fmt.Sprintf("Hosts[%d]", i), "BMC", "Address")
-				logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, host.BMC.Address))
-			}
-			if host.BMC.DisableCertificateVerification {
-				fieldPath := field.NewPath("Platform", "Baremetal", fmt.Sprintf("Hosts[%d]", i), "BMC", "DisableCertificateVerification")
-				logrus.Warnf(fmt.Sprintf("%s: true is ignored", fieldPath))
-			}
 			// The default is UEFI. +kubebuilder:validation:Enum="";UEFI;UEFISecureBoot;legacy. Set from generic install config code
 			if host.BootMode != "UEFI" {
```

Comment:
Can we not set the BootMode in the install-config overrides?

Reply:
We could, but this wasn't one of the fields included in the design since this bootmode would only be relevant in day 2. 
Do we want to add this?  If so, I'll create a bug and add it as a follow-on.